core/gesture.js

```diff
@@ -1004,6 +1004,7 @@ Blockly.Gesture.prototype.duplicateOnDrag_ = function() {
       newVariableField.textContent = xmlBlockField.textContent;
       newVariableBlock.appendChild(newVariableField);
       xmlBlock = newVariableBlock;
+      this.targetBlock_.inputList[0].fieldRow[0].clearHover();
```

Comment:
No guards needed here?

Reply:
This code path is only for that one block. Lots of assumptions here.